internal/pipe/nfpm/nfpm.go

```diff
@@ -338,6 +360,16 @@ func create(ctx *context.Context, fpm config.NFPM, format string, artifacts []*a
 		}
 	}
 
+	// If platform is manually set, prefer it
+	if len(fpm.Platform) > 0 {
+		infoPlatform = fpm.Platform
+	}
+
+	// If rpmArch is manually set, prefer it
+	if len(overridden.RPM.Arch) > 0 {
+		rpmArch = overridden.RPM.Arch
+	}
+
```

Comment:
i wonder how needed these really are

Reply:
With another look at the way overrides are handled, the rpmArch may not be needed. I'm not familiar enough with the codebase to be 100% certain. I've always taken the approach of "better safe than sorry", and remove it later if feedback says it isn't needed.

That said, it was done because of this:
https://github.com/goreleaser/goreleaser/pull/5075/files#diff-6cbd845fb048a5924cf8ea025ab328901f9b3efcb79523861a512472a42d42c1R195

Trying to set "intelligent" defaults for AIX, if someone skips setting the platform and arch.

Reply:
yeah, I think I might remove it.


to clarify, AIX is only ever ppc, right?

Reply:
ah, gotcha... so I think we can probably remove these two new options (for now, at least), and handle it gracefully

Reply:
also, Go's seems to only support `aix ppc64` 🤔

Reply:
Right, Go only supports recent versions of AIX, which are all ppc64 for goarch, but need to be ppc for rpm arch due to AIX weirdness

Reply:
yeah got that, its being done here :)